internal/course/database/course_queries.go

```diff
@@ -1,27 +1,39 @@
 package database
 
+import (
+	"fmt"
+)
+
 const (
-	createCourse       = "create course"
-	deleteCourse       = "delete course by uuid"
-	getCourse          = "get course by uuid"
-	listCourse         = "list course"
-	updateCourseByID   = "update course by uuid"
-	updateCourseByCode = "update course by code"
+	returningColumns = "uuid, code, name, underline, image, image_cover, excerpt, description, created_at, updated_at"
+	// CREATE.
```

Comment:
Shoud we return delet_at column? @rluders

Reply:
Yep

Reply:
As we are using the "deleted_at" column for soft deletion, I believe that our CRUD queries should specifically filter for records where "deleted_at" is null. This ensures that we only interact with active records during regular operations, maintaining the integrity and relevance of our data. Additionally, I propose that we create a dedicated endpoint to retrieve all records, including those marked as deleted. This endpoint would include the "deleted_at" information in its response, allowing for a comprehensive view of the data, both active and deleted. This approach enables us to keep the regular data interactions clean and focused while still providing the capability to audit or review the entire dataset when needed. What do you think?

Reply:
I don't see the need for 2 endpoints, I believe it could be filtered data.

Reply:
I'm going to create an issue to discuss and implement solutions related to recovering and listing deleted data, ok? @arianerocha @rluders